sunflower-land/sunflower-land
# [FEAT] Upgrade Island

# Description

This PR includes the logic to upgrade your island - `farm.upgraded`

<img width="272" alt="Screenshot 2024-01-17 at 10 49 08 am" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/d7eebbb5-3cbf-4f6e-af2f-ff6e0ebee1ba">
<img width="570" alt="Screenshot 2024-01-17 at 10 49 11 am" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/5d3875fd-d45b-400a-a1af-3e00692a9ae3">

**Excludes**

- Exact requirements
- Expanding on new island + resources

## How to test?

1. Go to Grubnuk on the raft
2. Confirm Upgrade
